### PR TITLE
Make it possible to accept the terms and conditions

### PR DESCRIPTION
**Pull request by Pilzington**

### CODE REVIEW

File: proto/api/v1/tenant.proto [hunk 1]
```diff
@@ -71,6 +72,16 @@ message PaymentDetailsUpdate {
   optional string subscription_id = 3;
 }
 
+message AcceptedTermsAndConditionsDetails {
```
Comment (majst01) [inline proto/api/v1/tenant.proto:75]: Simply TermsAndConditions ? other Message also ?

Comment (Pilzington) [reply proto/api/v1/tenant.proto:75]: Not sure what you mean. I try to implement this: https://github.com/metal-stack-cloud/console/issues/288  
Just save in masterdata if the user accepted the terms and when he did it.

Comment (majst01) [reply proto/api/v1/tenant.proto:75]: Shorter name of the Struct i mean `TermsAndConditions` should be enough if this has two fields `accepted` and `when`

Comment (Pilzington) [reply proto/api/v1/tenant.proto]: done

**Pull request merged**